# THE ISOFORM POTENTIAL OF THE GENOME

While the diversity of cellular functions is attributed, in part, to the protein isoform variation mediated by pre-mRNA splicing, a comprehensive catalog of candidate functional isoforms is lacking. An atlas of predicted alternative splicing events with functional potential â€“ intact, conserved open-reading frames (ORF) - would provide a platform for protein isoform discovery, detection and characterization in physiology and disease states.   

Protein isoforms produced from alternative splicing events of specific genes are of major interest to the Hancks Lab for their potential to expand and diversify the host arsenal against invading pathogens and regulate immune responses. Spliceform discovery has been driven by cDNA analysis. However, this framework is limited in detection power when spliceforms are: 1) tissue- or cell-type (unsampled), 2) inducible, 3) developmentally-regulated, or 4) low abundance variants. We propose to develop a bioinformatic framework that could serve to identify spliceform candidates genome-wide and seed biological investigation.

To define the landscape of potential ORFs from, nonsense-mediated decay (NMD) pathway rules should be used as an initial filter. To further limit false positives, sequence across species conservation can be exploited. To aid in the characterization of predicted isoforms, gene ontology analysis should be performed. The utility of this information would be maximized if translated into a genome browser track allowing accessibility to the community at large and those interested in the biology of specific-genes/pathways. Described filtering would require incorporating either scripts and/or existing databases for translation of DNA sequences, NMD predicted-susceptibility, and gene ontology analysis.

Team Lead: Dustin Hancks, Immunology, www.utsouthwestern.edu/labs/hancks/ 
